cicd/common.sh

```diff
@@ -13,6 +13,8 @@ hns="sudo ip netns "
 hexist="$vrn$hn"
 lxdocker="ghcr.io/loxilb-io/loxilb:latest"
 cluster_opts=""
+## FIXME - This is needed for perfCI scripts
+OSE_LOXILB_SERVERS=${OSE_LOXILB_SERVERS:=1}
```

Comment:
With https://github.com/loxilb-io/loxilb/pull/460 this should no longer be required.